3.3.0 test failure in collection

collected 356 items / 1 error                                                                   

============================================ ERRORS =============================================
________________________ ERROR collecting examples/save_class_to_file.py ________________________
/usr/lib/python3.12/site-packages/_pytest/runner.py:341: in from_call
    result: Optional[TResult] = func()
/usr/lib/python3.12/site-packages/_pytest/runner.py:372: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/usr/lib/python3.12/site-packages/pytest_asyncio/plugin.py:644: in _patched_collect
    module = collector.obj
/usr/lib/python3.12/site-packages/_pytest/python.py:310: in obj
    self._obj = obj = self._getobj()
/usr/lib/python3.12/site-packages/_pytest/python.py:528: in _getobj
    return self._importtestmodule()
/usr/lib/python3.12/site-packages/_pytest/python.py:617: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/usr/lib/python3.12/site-packages/_pytest/pathlib.py:567: in import_path
    importlib.import_module(module_name)
/usr/lib64/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1387: in _gcd_import
    ???
<frozen importlib._bootstrap>:1360: in _find_and_load
    ???
<frozen importlib._bootstrap>:1331: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:935: in _load_unlocked
    ???
<frozen importlib._bootstrap_external>:995: in exec_module
    ???
<frozen importlib._bootstrap>:488: in _call_with_frames_removed
    ???
examples/save_class_to_file.py:27: in <module>
    encoded_instance
E   AssertionError
==================================== short test summary info ====================================
ERROR examples/save_class_to_file.py - AssertionError
!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================= 1 error in 4.92s ========================================

--- a/conftest.py
+++ b/conftest.py
@@ -1,4 +1,4 @@
-collect_ignore = ['contrib']
+collect_ignore = ['contrib', 'examples']
 
 
 def pytest_addoption(parser):

